app/views/layouts/listnav/_ems_cloud.html.haml

```diff
@@ -40,7 +40,7 @@
             :tables        => 'flavor',
             :controller    => 'ems_cloud')
         - if role_allows(:feature => "security_group_show_list")
-          = li_link(:count => @record.number_of(:security_groups),
+          = li_link(:count => @record.security_groups && @record.number_of(:security_groups),
```

Comment:
I'll start by saying that I was including it only because it was in @Ladas' code.  I didn't even know it existed before this. 

But, what's worrying about it?  [It looks pretty simple](https://github.com/ManageIQ/manageiq/blob/85fdc0a5bdd96d8e39d080d95c0fd3389a3b78bf/lib/extensions/ar_number_of.rb).

It's strange that we'd cache the count like that, but in the time it takes for the request to complete and this AR object to come into and out of existence, that count probably won't change, I guess.  And, if it does, it's arguable whether it's "wrong" at the time the page is built anyway.